intro-HPC/ch_introduction.tex

```diff
@@ -257,31 +297,36 @@ \subsection{What programming languages can I use?}
 Commonly used software packages are ABINIT, CP2K, Gaussian, Gromacs, Molpro,
 NWChem, Quantum Espresso, R, Siesta, TURBOMOLE, WIEN2k,\ldots
 
-Commonly used Libraries are Intel MKL, FFTW, HDF5, PETSc and Intel MPI, M(VA)PICH, OpenMPI.
+Commonly used Libraries are Intel MKL, FFTW, HDF5, PETSc and Intel MPI,
+M(VA)PICH, OpenMPI.
 \fi
 \ifleuven
 Commonly used software packages are ABINIT, CP2K, Gaussian, Gromacs, Molpro,
 NWChem, Quantum Espresso, R, Siesta, TURBOMOLE, WIEN2k,\ldots
 
-Commonly used Libraries are Intel MKL, FFTW, HDF5, PETSc and Intel MPI, M(VA)PICH, OpenMPI.
+Commonly used Libraries are Intel MKL, FFTW, HDF5, PETSc and Intel MPI,
+M(VA)PICH, OpenMPI.
 \fi
 \ifbrussel
 Commonly used software packages are CP2K, Gaussian, MATLAB, NWChem, R,\ldots
 
-Commonly used Libraries are Intel MKL, FFTW, HDF5, netCDF, PETSc and Intel MPI, OpenMPI.
+Commonly used Libraries are Intel MKL, FFTW, HDF5, netCDF, PETSc and Intel MPI,
+OpenMPI.
 \fi
 
-Additional software can be installed ``\emph{on demand}''. Please check the
+Additional software can be installed ``\emph{on demand}''. Please contact the
 \hpc staff to see whether the \hpc can handle your specific requirements.
 
 \subsection{What operating systems can I use?}
 \label{sec:what-operating-systems-can-i-use}
 
-All nodes in the \hpc cluster run under \operatingsystemSL, which is a specific version of \operatingsystemRHEL. This means that
-all programs (executable) must be compiled for \operatingsystemSL.
+All nodes in the \hpc cluster run under \operatingsystemSL, which is a specific
+version of \operatingsystemRHEL. This means that all programs (executable) must
```

Comment:
the `must` makes no sense. `should` is better
and executable should be plural (`programs` -> `executables`)